util/src/main/java/tech/pegasys/artemis/util/config/Eth1DepositContractAddress.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.util.config;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import java.util.Objects;
+import org.apache.tuweni.bytes.Bytes;
+
+public class Eth1DepositContractAddress {
+  /** The number of bytes in this value - i.e. 20 */
+  private static final int SIZE = 20;
+
+  private final Bytes bytes;
+
+  public Eth1DepositContractAddress(Bytes bytes) {
+    checkArgument(
+        bytes.size() == SIZE,
+        "Bytes%s should be %s bytes, but was %s bytes.",
+        SIZE,
+        SIZE,
+        bytes.size());
+    this.bytes = bytes;
+  }
+
+  @Override
+  public boolean equals(final Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    final Eth1DepositContractAddress address = (Eth1DepositContractAddress) o;
+    return bytes.equals(address.bytes);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(bytes);
+  }
+
+  @Override
+  public String toString() {
+    return bytes.toString();
+  }
+
+  public static Eth1DepositContractAddress fromHexString(String value) {
+    Bytes bytes = Bytes.fromHexString(value);
+    return new Eth1DepositContractAddress(bytes);
+  }
+
+  public String toHexString() {
+    return bytes.toHexString();
+  }
+
+  public Bytes toBytes() {
+    return bytes;
+  }
+
+  public static Eth1DepositContractAddress empty() {
+    return new Eth1DepositContractAddress(Bytes.wrap(new byte[SIZE]));
+  }
```

Comment:
nit: Probably better to make this a constant since the class is immutable.

Reply:
we're not using empty at the moment, so I'll just remove it.